<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scenario</string>
  </property>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save Scenario</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="general_settings_group">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>620</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>General  settings</string>
   </property>
   <widget class="QLabel" name="model_time_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Modelling Time, minutes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="model_time_line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="min_dt_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Generation Timestep, seconds</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_dt_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sensors_group">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>210</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Sensors parameters</string>
   </property>
   <widget class="QPushButton" name="sensors_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sensors Table</string>
    </property>
   </widget>
   <widget class="QLabel" name="sensors_total_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sensors total</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sensors_total_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_as_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save Scenario As</string>
   </property>
  </widget>
  <widget class="QPushButton" name="revert_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Revert Changes</string>
   </property>
  </widget>
  <widget class="QLabel" name="motion_seed_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Random Seed</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="route_group">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>210</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>Base route parameters</string>
   </property>
   <widget class="QLabel" name="straight_section_time_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Straight section time range, min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_straight_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_straight_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="min_straight_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_straight_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLabel" name="rot_radius_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Maneuvers radius range, kn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_radius_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="min_radius_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_radius_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="max_radius_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_range_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Speed range, kn</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_speed_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="min_speed_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_speed_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="max_speed_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="zero_start_speed_checkbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Zero start speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_prob_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Acceleration probability</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="acc_prob_line">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="wave_group">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>390</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Random wave parameters</string>
   </property>
   <widget class="QSlider" name="wave_period_slider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
    <property name="sliderPosition">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSlider" name="wave_height_slider">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
    <property name="sliderPosition">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="wave_period_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="wave_height_line">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="wave_period_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wave period, s</string>
    </property>
   </widget>
   <widget class="QLabel" name="wave_height_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wave height, m</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="motion_seed_line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="geo_location_group">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>210</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>Geographic Location</string>
   </property>
   <widget class="QLabel" name="ellipsoi_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Ellipsoid</string>
    </property>
   </widget>
   <widget class="QLabel" name="height_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Height, m</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="height_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="latitude_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Latitude, deg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="latitude_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="longitude_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Longitude, deg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="longitude_line">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ellipsoid_combo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>CGCS2000 (BeiDou)</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>WGS84 (GPS)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ПЗ-90.02 (GLONASS)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CGCS2000 (BeiDou)</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="attitude_group">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>160</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Attitude Angles</string>
   </property>
   <widget class="QLabel" name="course_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Course, deg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="course_line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="roll_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Roll, rad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roll_line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pitch_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pitch, rad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pitch_line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="srns_loss_group">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>390</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>SRNS group loss parameters</string>
   </property>
   <widget class="QLineEdit" name="min_loss_time_line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_loss_time_line">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loss_time_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Loss time, minutes</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_loss_time_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_loss_time_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QLabel" name="loss_prob_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Loss probability</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loss_prob_line">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
